packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [23.0.0]
+
+### Changed
```

Comment:
I think Added and Changed need to be flipped — that's why you're getting the lint error.

Reply:
winner